src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java

```diff
@@ -71,7 +71,7 @@ public Item createGun(CommonProxy commonProxy) {
         ))
 		.withInformationProvider(stack -> Arrays.asList(
 		"Type: .50 Beowulf Rifle", 
-		"Damage: 7", 
+		"Damage: 10",
```

Comment:
please revert this change (I will do extended research on balance of weapons at a later date)